content/agent/faq/rbac-for-dca-running-on-aks-with-helm.md

```diff
@@ -0,0 +1,33 @@
+Does your Cluster Agent restart all the time on AKS?
+
+It might be because you have deployed it with our helm chart and the readiness probe is killing it.
```

Comment:
Maybe something like "Depending on how you deployed the manifest, you might be using readiness probes. On Azure AKS you will need extra permissions in the Cluster Role in order for the health endpoint to be reachable."

Reply:
Remove the word `will` - no future tense in docs:

"Depending on how you deployed the manifest, you might be using readiness probes. On Azure AKS you need extra permissions in the Cluster Role in order for the health endpoint to be reachable."